### PR TITLE
Feature/add psi

### PR DESCRIPTION
**Pull request by Piranias**
perso-si timeseries integrated

- [x] add timeseries calculation

- [x] add calculation nominal value

### CODE REVIEW

**Comment by Piranias** [comment]
Hi @SabineHaas,
I updated the greco_technologies repo with a new folder "perosi". I added both, the CPV_dev and the perosi_dev to a common "dev" branch.
I only changed the requirements in pvcompare from CPV_dev to dev but still the module cannot be found. Do I need to change something else in pvcompare so that it's using the dev branch from now on?

**Comment by SabineHaas** [comment]
No that looks alright! Can you run the test locally? Just to be sure that there are no spelling errors.. 
You can also try to locally install the requirements.txt (pip install -r path/to/requirements) in a clean virtual environment and try to run the test there.

**Comment by Piranias** [comment]
> No that looks alright! Can you run the test locally? Just to be sure that there are no spelling errors..
> You can also try to locally install the requirements.txt (pip install -r path/to/requirements) in a clean virtual environment and try to run the test there.

so now the greco-technologies.perosi folder is found. But the packages used in this repository (like the feedinlib) is not found. I added it to the setup.py in greco-technologies and tries a lot, but nothing seems to work.

File: requirements.txt [hunk 1]
```diff
@@ -14,5 +14,5 @@ demandlib
 oemof
 pdfkit==0.6.1
 xarray==0.15.0
-feedinlib
+feedinlib==0.0.12
```
Comment (SabineHaas) [inline requirements.txt:18]:
```suggestion
feedinlib==0.1.0rc2
```

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,7 @@
 git+https://github.com/greco-project/greco_technologies.git@dev
 git+https://github.com/greco-project/cpvtopvlib.git@master
 git+https://github.com/rl-institut/mvs_eland.git@dev
+git+https://github.com/oemof/feedinlib.git@dev
```
Comment (SabineHaas) [inline requirements.txt:4]:
```suggestion

```

**Comment by SabineHaas** [comment]
> > No that looks alright! Can you run the test locally? Just to be sure that there are no spelling errors..
> > You can also try to locally install the requirements.txt (pip install -r path/to/requirements) in a clean virtual environment and try to run the test there.
> 
> so now the greco-technologies.perosi folder is found. But the packages used in this repository (like the feedinlib) is not found. I added it to the setup.py in greco-technologies and tried a lot, but nothing seems to work.

It was the version of the feedinlib - only the newest tag contains the era5 functionalities.

**Comment by Piranias** [comment]
> > > No that looks alright! Can you run the test locally? Just to be sure that there are no spelling errors..
> > > You can also try to locally install the requirements.txt (pip install -r path/to/requirements) in a clean virtual environment and try to run the test there.
> > 
> > 
> > so now the greco-technologies.perosi folder is found. But the packages used in this repository (like the feedinlib) is not found. I added it to the setup.py in greco-technologies and tried a lot, but nothing seems to work.
> 
> It was the version of the feedinlib - only the newest tag contains the era5 functionalities.

great, thanks, I tried with the current dev version because it also contains the era5 module. But at least now its working :)

**Comment by SabineHaas** [comment]
Oh, I didn't do a review, yet. I saw strangely formatted characters in the changes, mostly where units are written. Can you check whether they are fine? Also I think the psi time series is not calculated, yet.

File: pvcompare/pv_feedin.py [hunk 3]
```diff
@@ -298,10 +300,10 @@ def set_up_system(technology, surface_azimuth, surface_tilt, cpv_type):
         return cpv_sys, module_params
 
     elif technology == "psi":
-        raise ValueError("The nominal value for psi cannot be calculated yet.")
+        pass
```
Comment (SabineHaas) [inline pvcompare/pv_feedin.py:303]: Shouldn't the time series be calculated here?

Comment (Piranias) [reply pvcompare/pv_feedin.py:303]: no, this is where the classes for si and cpv and the module_parameters dictionaries are being set up. The psi-timeseries "skips" this part and is calculated [here](https://github.com/greco-project/pvcompare/blob/0cf406d551646fd78bf4e92318b92e7818c67456/pvcompare/pv_feedin.py#L149)

**Comment by Piranias** [comment]
> Oh, I didn't do a review, yet. I saw strangely formatted characters in the changes, mostly where units are written. Can you check whether they are fine? Also I think the psi time series is not calculated, yet.

sorry, I thought it was just this error missing. Do you want to unmerge? I'm not sure how this is done..

**Comment by SabineHaas** [comment]
No worries, no let's keep it merged! I hadn't reviewed further because I wasn't sure whether you were ready for review already., sorry! Could you adjust the mis-formatted characters in the other modules still? Probably it was an encoding error.

**Pull request merged**